python/lsst/metaserv/schemaToMeta.py

```diff
@@ -47,7 +47,7 @@ class SchemaToMeta(object):
 
     _tableStart = re.compile(r'CREATE TABLE (\w+)*')
     _tableEnd = re.compile(r"\)")
-    _engineLine = re.compile(r'\) (ENGINE|TYPE)=(\w+)*;')
+    _engineLine = re.compile(r'\)[\s]*(ENGINE|TYPE)[\s]*=[\s]*(\w+)*;')
```

Comment:
No need for brackets if there's only one item within them.
`(\w+)*` is the same as `(\w*)`, but I don't think you want that; I think you want `(\w+)\s*`.